libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/InstructionModel.java

```diff
@@ -12,10 +15,21 @@
   private String drivingSide;
 
   public InstructionModel(DistanceFormatter distanceFormatter, RouteProgress progress) {
```

Comment:
As this is a `public` method, could we add `@Nullable` / `@NonNull` annotations to help Kotlin users?

Reply:
I think it should be `@Nullable`. Considering the Free Drive mode, routeProgress can be null.

Reply:
What about `DistanceFormatter`?